docs/_includes/generated/core_configuration.html

```diff
@@ -22,6 +22,16 @@
             <td style="word-wrap: break-word;">"child-first"</td>
             <td>Defines the class resolution strategy when loading classes from user code, meaning whether to first check the user code jar ("child-first") or the application classpath ("parent-first"). The default settings indicate to load classes first from the user code jar, which means that user code jars can include and load different dependencies than Flink uses (transitively).</td>
         </tr>
+        <tr>
+            <td><h5>debug.system-resource-metrics</h5></td>
```

Comment:
How about moving these to `MetricOptions`? (they could also have a "metrics" prefix instead)